modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/TeamMembers/index.js

```diff
@@ -28,11 +30,17 @@ const InvitesModal = ({observer, onClose, project}) => {
 };
 
 const TeamMembers = () => {
-	const {project} = useOutletContext();
+	const {project, subscriptionGroups} = useOutletContext();
 	const [visible, setVisible] = useState(false);
 	const modalProps = useModal({
 		onClose: () => setVisible(false),
 	});
+	const [activatedStatusDXPC, setActivatedStatusDXPC] = useState();
+
+	useEffect(() => {
+		const activationStatusDXPC = subscriptionGroups[0].activationStatus;
+		setActivatedStatusDXPC(activationStatusDXPC);
+	}, [subscriptionGroups]);
```

Comment:
It's not necessary add a state + useffect to get a value that can be updated automatically

```suggestion
	const activatedStatusDXPC = subscriptionGroups[0]?.activationStatus;
```

Reply:
Done.